### PR TITLE
[Don't Merge This] -- Show merge conflicts between 0.6.0 and master

### PR DESCRIPTION
**Pull request by isnotinvain**

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -46,9 +46,15 @@ object SummingbirdBuild extends Build {
 
   val sharedSettings = extraSettings ++ Seq(
     organization := "com.twitter",
+<<<<<<< HEAD
```
Comment (isnotinvain) [inline project/Build.scala]: Everything in this file is easy to fix, so no worries here

File: summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java [hunk 2]
```diff
@@ -82,6 +82,10 @@ public Void apply(IN v) {
     return new Service<Memory, JMemoryService<K, V>, K, V>(new JMemoryService(service));
   }
 
+<<<<<<< HEAD
+=======
+  private Memory platform = new Memory(new JobId("inMem"));
```
Comment (isnotinvain) [inline summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java]: was this deleted on develop on purpose, or added in master and we want to keep it?

File: summingbird-core/src/main/scala/com/twitter/summingbird/Stats.scala [hunk 3]
```diff
@@ -93,11 +93,16 @@ object JobCounters {
   private val registeredCountersForJob: ConcurrentHashMap[JobId, ParHashSet[(Group, Name)]] =
     new ConcurrentHashMap[JobId, ParHashSet[(Group, Name)]]()
 
+<<<<<<< HEAD
```
Comment (isnotinvain) [inline summingbird-core/src/main/scala/com/twitter/summingbird/Stats.scala]: Assuming we just want the HEAD block here? (head is the release branch)

File: summingbird-core/src/main/scala/com/twitter/summingbird/memory/Memory.scala [hunk 4]
```diff
@@ -41,8 +47,15 @@ class Memory(implicit jobID: JobId = JobId("default.memory.jobId")) extends Plat
   private type Prod[T] = Producer[Memory, T]
   private type JamfMap = Map[Prod[_], Stream[_]]
 
+<<<<<<< HEAD
```
Comment (isnotinvain) [inline summingbird-core/src/main/scala/com/twitter/summingbird/memory/Memory.scala]: assuming HEAD is more up to date here too?

File: summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java (hunk 2 above)
Comment (johnynek) [inline summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java]: It was added, if I'm not mistaken.

Comment (johnynek) [inline summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java]: Wait, actually, I don't know.

Comment (johnynek) [inline summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java]: @egonina should comment here. I guess we are going to want it.

File: summingbird-core/src/main/scala/com/twitter/summingbird/memory/Memory.scala (hunk 4 above)
Comment (egonina) [inline summingbird-core/src/main/scala/com/twitter/summingbird/memory/Memory.scala]: Yes, HEAD is the latest

File: summingbird-core/src/main/scala/com/twitter/summingbird/Stats.scala (hunk 3 above)
Comment (egonina) [inline summingbird-core/src/main/scala/com/twitter/summingbird/Stats.scala]: Yes

File: summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java (hunk 2 above)
Comment (egonina) [inline summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java]: No, we fixed it with [this](https://github.com/twitter/summingbird/blob/develop/summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java#L33) in the later version. We can take the HEAD version here.

**Pull request closed without merging**